<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTCH-5CD105HTG9" name="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles" tests="18" failures="3" timestamp="9 Apr 2021 05:53:26 GMT" time="29.241" errors="0">
  <testcase name="postCreateAnEngagementRoleForAnOrganization_status409" time="1.041" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="postCreateAnEngagementRoleForAMethodology_status409" time="1.605" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles">
    <failure type="java.lang.AssertionError" message="expected [409] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [409] but found [200]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles.postCreateAnEngagementRoleForAMethodology_status409(Restassured_Automation_EngagementTeamRoles.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- postCreateAnEngagementRoleForAMethodology_status409 -->
  <testcase name="patchUpdateAnEngagementRoleForAnOrganization_status204" time="1.531" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="postCreateAnEngagementRoleForAMethodology_status400" time="1.565" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="deleteAnEngagementRoleForAMethodology_status204" time="2.362" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="postCreateAnEngagementRoleForAMethodology_status200" time="1.582" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="getLoadAllEngagementRolesInformationForARevision_status200" time="1.625" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="postCreateAnEngagementRoleForAnOrganization_status200" time="1.003" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="patchUpdateAnEngagementRoleForAnOrganization_status409" time="1.512" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles">
    <failure type="java.lang.AssertionError" message="expected [409] but found [204]">
      <![CDATA[java.lang.AssertionError: expected [409] but found [204]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles.patchUpdateAnEngagementRoleForAnOrganization_status409(Restassured_Automation_EngagementTeamRoles.java:297)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- patchUpdateAnEngagementRoleForAnOrganization_status409 -->
  <testcase name="deleteDeleteAnEngagementRoleForAnOrganization_status400" time="1.483" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="getLoadAllEngagementRolesInformationForAnOrganization_status200" time="0.983" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="patchUpdateAnEngagementRoleForAnOrganization_status400" time="1.466" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="patchUpdateAnEngagementRoleForAMethodology_status400" time="2.059" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="patchUpdateAnEngagementRoleForAMethodology_status204" time="2.128" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="postCreateAnEngagementRoleForAnOrganization_status400" time="0.981" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="patchUpdateAnEngagementRoleForAMethodology_status409" time="2.151" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles">
    <failure type="java.lang.AssertionError" message="expected [409] but found [204]">
      <![CDATA[java.lang.AssertionError: expected [409] but found [204]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles.patchUpdateAnEngagementRoleForAMethodology_status409(Restassured_Automation_EngagementTeamRoles.java:773)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- patchUpdateAnEngagementRoleForAMethodology_status409 -->
  <testcase name="deleteAnEngagementRoleForAMethodology_status400" time="2.671" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
  <testcase name="deleteDeleteAnEngagementRoleForAnOrganization_status204" time="1.493" classname="restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles"/>
</testsuite> <!-- restassured.automation.testcases.Restassured_Automation_EngagementTeamRoles -->
